Documentation/ApiOverview/BackendLayout/Index.rst

```diff
@@ -0,0 +1,211 @@
+
+.. code-block:: typoscript
+
+   mod.web_layout.BackendLayouts {
+     exampleKey {
+       title = Example
+       icon = EXT:example_extension/Resources/Public/Images/BackendLayouts/default.gif
+       config {
+         backend_layout {
+           colCount = 3
+           rowCount = 3
+           rows {
+             1 {
+               columns {
+                 1 {
+                   name = Header
+                   colspan = 3
+                   colPos = 1
+                 }
+               }
+             }
+             2 {
+               columns {
+                 1 {
+                   name = Main
+                   colspan = 2
+                   colPos = 0
+                 }
+                 2 {
+                   name = Aside
+                   rowspan = 2
+                   colPos = 2
+                 }
+               }
+             }
+             3 {
+               columns {
+                 1 {
+                   name = Main Left
+                   colPos = 5
+                 }
+                 2 {
+                   name = Main Right
+                   colPos = 6
+                 }
+               }
+             }
+           }
+         }
+       }
+     }
+   }
+
+
+.. _be-layout-frontend:
+
+Output of a Backend Layout in the frontend
+==========================================
+
+The backend layout to be used on a certain page gets determined either by the backend layout being chosen directly and
+stored in the pages field "backend_layout" or by the field "backend_layout_next_level" of a parent page up the rootline
+
+To avoid complex TypoScript for integrators, the handling of backend layouts has
+been simplified for the frontend.
+
+To get the correct backend layout, the following TypoScript code can be used:
+
+.. code-block:: typoscript
+
+	page.10 = FLUIDTEMPLATE
+	page.10 {
+	  file.stdWrap.cObject = CASE
+	  file.stdWrap.cObject {
+		key.data = pagelayout
+
+		default = TEXT
+		default.value = EXT:sitepackage/Resources/Private/Templates/Home.html
+
+		3 = TEXT
+		3.value = EXT:sitepackage/Resources/Private/Templates/1-col.html
+
+		4 = TEXT
+		4.value = EXT:sitepackage/Resources/Private/Templates/2-col.html
+	  }
+	}
+
+Using  `data = pagelayout` is the same as using as
+
+.. code-block:: typoscript
+
+	field = backend_layout
+	ifEmpty.data = levelfield:-2,backend_layout_next_level,slide
+	ifEmpty.ifEmpty = default
+
+In the fluid template the column positions can be accessed now via content mapping as described here
+:ref:`t3sitepackage:content-mapping`.
+
+
+.. _be-layout-extensions:
+
+Extensions for the Backend Layouts
+==================================
+
+In many cases beside defining fixed Backend Layouts a more modular approach may be feasible Where different layouts in
+Backend and Frontend can be combined. The extension `gridelements <https://extensions.typo3.org/extension/gridelements/>`__
+integrates the grid layout concept also to regular content elements.
```

Comment:
I would write: "In many cases besides defining fixed backend layouts a more modular approach with the possibility of combining different backend layouts and frontend layouts may be feasible."
Also check the spelling (words except for names should be lower case)